### PR TITLE
fetching movie details works

### PR DESCRIPTION
**Pull request by ChopperTheKing**
image

### CODE REVIEW

**Review by k-code00 — APPROVED** [review]
did you design your own UI for this?

File: CineSightApp/CineSightApp/MovieService.swift [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+                            completion(.failure(decodeError))
+                        }
+                    }
+                }
+            }.resume()
+        }
+    }
+}
+
+struct SearchResponse: Codable {
```
Comment (mattmaloof) [inline CineSightApp/CineSightApp/MovieService.swift]: use Decodable.

File: CineSightApp/CineSightApp/Movie.swift [hunk 2]
```diff
@@ -0,0 +1,15 @@
+//
+//  Movie.swift
+//  CineSightApp
+//
+//  Created by Ronnie Kissos on 10/5/23.
+//
+
+import Foundation
+
+struct Movie: Codable {
```
Comment (mattmaloof) [inline CineSightApp/CineSightApp/Movie.swift]: Use Decodable.

**Review by mattmaloof — CHANGES_REQUESTED** [review]
change to Decodable where indicated. otherwise is fine.

**Comment by k-code00** [comment]
can we also adhere to the PR naming convention?

**Review by gisele-prog — APPROVED** [review]
looks fine

**Review by ChopperTheKing — COMMENTED** [review]
Changed from codable to decodable

**Review by ChopperTheKing — COMMENTED** [review]
Fixed decodable/codable thing.

**Review by mattmaloof — APPROVED** [review]

**Pull request merged**